### PR TITLE
COMMERCE-9614 Fixed product page templates

### PR DESCRIPTION
**Pull request by alessiorendina**
https://issues.liferay.com/browse/COMMERCE-9614

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemObjectProvider.java [hunk 1]
```diff
@@ -51,16 +53,16 @@ public CPDefinition getInfoItem(InfoItemIdentifier infoItemIdentifier)
 		ClassPKInfoItemIdentifier classPKInfoItemIdentifier =
 			(ClassPKInfoItemIdentifier)infoItemIdentifier;
 
-		CPDefinition cpDefinition = _cpDefinitionLocalService.fetchCPDefinition(
-			classPKInfoItemIdentifier.getClassPK());
-
-		if (cpDefinition == null) {
+		try {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemObjectProvider.java:56]: why this was made that way? your version looks cleaner 👍

**Review by andrea-ale-sbarra — APPROVED** [review]

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemObjectProvider.java (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemObjectProvider.java:56]: I don't know why but without the throwable the exception was not handled correctly by the framework

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**